chainer/functions/activation/log_softmax.py

```diff
@@ -87,17 +87,16 @@ def backward(self, x, gy):
 
 
 def log_softmax(x, use_cudnn=True):
-    """Channelwise log-softmax function.
+    """Channel-wise log-softmax function.
 
     This function computes its logarithm of softmax along the second axis. Let
-    :math:`x = (x_1, x_2, \\dots, x_d)^{\\top}` be the d dimensional index
-    array and :math:`f(x_1, \\dots, x_d)` be the corresponding input array.
-    For each index :math:`x` in the input array, it computes the logarithm
-    of the probability :math:`\log p(x)` defined as
+    :math:`x = (x_1, x_2, \\dots, x_D)^{\\top}` be the D dimensional input
+    array.
+    For each input array :math:`x`, it computes the logarithm of the function
+    :math:`\log f(x)` defined as
```

Comment:
"logarithm of the function log f(x)" looks duplicated expression. "logarithm of the function f(x)" is more clear. So how about replace this with :math:`f(x)` ?